### DISCUSSION TITLE
Overlapping sound recording

### DISCUSSION THREAD

**Discussion by girayyagmur** [pyannote/pyannote-audio#1271] [Q&A]
In the mono audio recording where 2 people speak. There are overlapping sounds. I want to turn the voice of the two people in the text. How can I solve overlapping here?

**Comment by hbredin** [comment]
`pyannote.audio` does not do speech-to-text.

**Reply by girayyagmur** [reply]
I know you don't translate speech to text. I'm asking if I can separate the voices of 2 people who are overlapping in Pyannote.audio.

**Reply by hbredin** [reply]
This is not what you asked in the first place but ok…

pyannote.audio does not do speaker separation either. 

You might want to have a look at asteroid source separation toolkit.
